template-compose/gradle/libs.versions.toml

```
@@ -0,0 +1,149 @@
+[versions]
+androidCompileSdk = "35"
+androidMinSdk = "24"
+androidTargetSdk = "35"
+androidVersionCode = "1"
+androidVersionName = "0.37.0"
+
+accompanist = "0.30.1"
+chucker = "3.5.2"
+composeBom = "2023.09.02"
+composeNavigation = "2.5.3"
+core = "1.10.1"
+coreKtx = "1.6.1"
+datastore = "1.0.0"
+detekt = "1.21.0"
+gradle = "8.1.2"
+hilt = "2.48"
+hiltNavigation = "1.0.0"
+javaxInject = "1"
+junit = "4.13.2"
+kotest = "5.6.2"
+kotlin = "1.9.10"
+kotlinCoroutines = "1.7.1"
+kover = "0.7.3"
+lifecycle = "2.6.1"
+mockk = "1.13.5"
+moshi = "1.12.0"
+nimbleCommon = "0.1.1"
+okhttp = "4.11.0"
+retrofit = "2.9.0"
+robolectric = "4.10.2"
+security = "1.0.0"
+timber = "4.7.1"
+turbine = "0.13.0"
+
+[libraries]
+# AndroidX
+androidx-core = { group = "androidx.core", name = "core-ktx", version.ref = "core" }
+androidx-lifecycle-runtime = { group = "androidx.lifecycle", name = "lifecycle-runtime-ktx", version.ref = "lifecycle" }
+androidx-lifecycle-compose = { group = "androidx.lifecycle", name = "lifecycle-runtime-compose", version.ref = "lifecycle" }
+androidx-datastorePreferences = { group = "androidx.datastore", name = "datastore-preferences", version.ref = "datastore" }
+androidx-securityCrypto = { group = "androidx.security", name = "security-crypto", version.ref = "security" }
+
+# Compose
+compose-bom = { group = "androidx.compose", name = "compose-bom", version.ref = "composeBom" }
+compose-ui = { group = "androidx.compose.ui", name = "ui" }
+compose-ui-tooling = { group = "androidx.compose.ui", name = "ui-tooling" }
+compose-ui-tooling-preview = { group = "androidx.compose.ui", name = "ui-tooling-preview" }
+compose-material = { group = "androidx.compose.material", name = "material" }
+compose-navigation = { group = "androidx.navigation", name = "navigation-compose", version.ref = "composeNavigation" }
+accompanist-permissions = { group = "com.google.accompanist", name = "accompanist-permissions", version.ref = "accompanist" }
+
+# Hilt
+hilt-android = { group = "com.google.dagger", name = "hilt-android", version.ref = "hilt" }
+hilt-navigation = { group = "androidx.hilt", name = "hilt-navigation-compose", version.ref = "hiltNavigation" }
+hilt-compiler = { group = "com.google.dagger", name = "hilt-compiler", version.ref = "hilt" }
+javax-inject = { group = "javax.inject", name = "javax.inject", version.ref = "javaxInject" }
+
+# Kotlin
+kotlin-coroutines-core = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version.ref = "kotlinCoroutines" }
```

We have `androidx`, then this one should be `kotlinx` to differs from main `kotlin` version 🧑‍🍳 

```suggestion
kotlinx-coroutines-core = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version.ref = "kotlinxCoroutines" }
```